src/main/java/apoc/export/cypher/formatter/CypherFormat.java

```diff
@@ -0,0 +1,30 @@
+package apoc.export.cypher.formatter;
+
+/**
+ * @author AgileLARUS
+ *
+ * @since 16-06-2017
+ */
+public enum CypherFormat {
+
+	CREATE("create"), ADD_STRUCTURE("addStructure"), UPDATE_STRUCTURE("updateStructure"), UPDATE_ALL("updateAll");
+
+	private String value;
+
+	CypherFormat(String value) {
+		this.value = value;
+	}
+
+	public static CypherFormat fromString(String value) {
```

Comment:
if we don't do case insensitive, and we would fail on null anyway we can also just use CypherFormat.valueOf